tls-client-pelion/README.md

```diff
@@ -0,0 +1,96 @@
+# HTTPS File Download Example for Pelion TLS Client on Mbed OS
+
+This application downloads a file from an HTTPS server (os.mbed.com) and looks for a specific string in that file.
+
+## Getting started
+
+Set up your environment if you have not done so already. For instructions, refer to the [main readme](../README.md).
+
+You can also compile this example with the [Mbed Online Compiler](https://os.mbed.com/compiler/) by using [this project](https://os.mbed.com/teams/mbed-os-examples/code/mbed-os-example-tls-tls-client).
+
+## Requirements
+
+This example uses by default an Ethernet connection to the internet.
+It's possible to switch to another network interface by using [Easy Connect](https://github.com/ARMmbed/easy-connect/blob/master/README.md). The [Mbed OS Example Client](https://github.com/ARMmbed/mbed-os-example-client#application-setup) application shows an example of how to configure other network interfaces.
+
+The networking stack used in this example requires TLS functionality to be enabled on Mbed TLS. On devices where hardware entropy is not present, TLS is disabled by default. This would result in compile time or linking failures.
+
+To learn why entropy is required, read the [TLS Porting guide](https://docs.mbed.com/docs/mbed-os-handbook/en/latest/advanced/tls_porting/).
+
+## Monitoring the application
+
+__NOTE:__ Make sure that the network is functional before running the application.
+
+The output in the terminal window should be similar to this:
+
+```
+Starting mbed-os-example-tls/tls-client-pelion
+Using Mbed OS 5.6.3
+[EasyConnect] IPv4 mode
```

Comment:
Again, I think this is out of date. As far as I am aware, the example does not use easy-connect anymore

Reply:
@andresag01 since it has not yet been decided, if this PR is going to be merged, and because it is outside of the PR's scope, I would rather raise an issue and fix it in a separate PR (this issue is also present in the regular tls-client). What do you think?